### ISSUE TITLE
Combos break Docking

### ISSUE THREAD

**Issue by krumelmonster** [ocornut/imgui#2570]
Version: 4158cba1ffd338c46d52164c2be0762031e66f53
Branch: docking

**Back-end/Renderer/Compiler/OS**

Back-ends: imgui_impl_opengl3.cpp; imgui_impl_glfw.cpp; GLEW
Compiler: gcc 8.3.0
Operating System: GNU/Linux

Note: I unset `ImGuiConfigFlags_ViewportsEnable`, it won't work with i3 (as to expect with a tiling wm)

**My Issue/Question:**

Opening a combo in the demo if the containing window is docked will undock the window.

**Screenshots/Video**

Docking the demo window:
![2019-05-21-182312_535x322_scrot](https://user-images.githubusercontent.com/13542760/58113601-34d42380-7bf6-11e9-89a9-5763dffd185c.png)
About to click this Combo:
![2019-05-21-182322_535x322_scrot](https://user-images.githubusercontent.com/13542760/58113633-49182080-7bf6-11e9-833b-e21009490fbc.png)
I click the Combo, it expands and Dockings break :(
![2019-05-21-182326_535x322_scrot](https://user-images.githubusercontent.com/13542760/58113675-5d5c1d80-7bf6-11e9-90ce-0aca952f4be9.png)

```
// Line 866 of imgui_demo.cpp
        // Simplified one-liner Combo() API, using values packed in a single constant string
        static int item_current_2 = 0;
        ImGui::Combo("combo 2 (one-liner)", &item_current_2, "aaaa\0bbbb\0cccc\0dddd\0eeee\0\0");
```

**Comment by ocornut** [comment]
There was a bug in 4158cba which I think has been fixed in next commit 2e5860b.

**Comment by krumelmonster** [comment]
Oh sorry, I thought I was up to date, it's fixed indeed.
